test/langtools/jdk/javadoc/doclet/testNavigation/TestNavigation.java

```diff
@@ -201,57 +172,82 @@ public static class Y extends X {
                     package pkg1; public interface InterfaceWithNoMembers {
                     }""");
 
-        javadoc("-d", "out-3",
+        javadoc("-d", "out-navlinks",
```

Comment:
See preceding comment. I would recommend using the equivalent of `base.resolve("api:).toString()` here.